airsonic-main/src/test/java/org/airsonic/player/service/sonos/sonos_manuel_test_.http

```diff
@@ -0,0 +1,22 @@
+#======================================================================================================
+# For manuel test
+#
```

Comment:
The proper spelling is "manual".  The filename should be corrected to "sonos_manual_test.h" as well.